Documentation/AEPEdgeIdentity.md

```diff
@@ -0,0 +1,98 @@
+# Getting Started
+
+# Android Test App
+## Testing tips with Android advertising identifier
+In Android, users are opted-in to ad ID tracking by default. They can choose to opt out of tracking in Android Settings at the device level.
+Developers using ad ID should get ad ID from the API each time it is used, as permissions for ad tracking and/or the value of the ID itself may be changed at any time.
+- In older versions of Android, the ad ID opt-in/out is a toggle, where the existing ad ID value remains unchanged.
+
+[<img src="./assets/old_adid_setting_optin.png" alt="Old ad ID settings page - opt-in state" width="210"/>](./assets/old_adid_setting_optin.png)
+[<img src="./assets/old_adid_setting_optout_prompt.png" alt="Old ad ID settings page - opt-out prompt" width="210"/>](./assets/old_adid_setting_optout_prompt.png)
+[<img src="./assets/old_adid_setting_optout.png" alt="Old ad ID settings page - opt-out state" width="210"/>](./assets/old_adid_setting_optout.png)
+
+- In newer versions of Android, the ad ID opt-in/out is a delete, where the existing ad ID value is deleted and replaced by an all-zero ad ID until the ad ID is recreated by user selection.
+
+[<img src="./assets/new_adid_setting_optin.png" alt="New ad ID settings page - opt-in state" width="350"/>](./assets/new_adid_setting_optin.png)
```

Comment:
The images look bigger than the old optout screenshot above,  can we change it to smaller?  Width = 250 or 300 and see which one looks better?